lib/dashing/app.rb

```diff
@@ -120,16 +119,6 @@ def protected!
   end
 end
 
-Thin::Server.class_eval do
-  def stop_with_connection_closing
-    Sinatra::Application.settings.connections.dup.each(&:close)
-    stop_without_connection_closing
-  end
-
-  alias_method :stop_without_connection_closing, :stop
-  alias_method :stop, :stop_with_connection_closing
-end
-
```

Comment:
Yes, this seems weird. Maybe this is okay to remove...

I feel like Thin should already be managing those connections with a graceful shutdown path too...